internal/namingschema/client_server.go

```diff
@@ -0,0 +1,57 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023 Datadog, Inc.
+
+package namingschema
+
+import "fmt"
+
+// ClientServerSystem represents client-server systems or protocols to be used for client-server naming
+// schemas in this package.
+type ClientServerSystem string
```

Comment:
Any reason why we don't support version override for this system?

Reply:
I didn't find a real need for the existing integrations, but I can add it everywhere for consistency 👍